build/AzurePipelinesTemplates/MUX-BuildProject-Steps.yml

```diff
@@ -0,0 +1,55 @@
+parameters:
+  solutionPath: ''
+  nugetConfigPath: ''
+  appxPackageDir: '$(build.artifactStagingDirectory)\$(buildConfiguration)\$(buildPlatform)\AppxPackages'
+  buildOutputDir: '$(Build.SourcesDirectory)\BuildOutput'
+  copyFilesToStaging: false
+  publishPath: '$env:Build_ArtifactStagingDirectory'
+
+steps:
+  - task: CmdLine@1
+    displayName: 'Display build machine environment variables'
+    inputs:
+      filename: 'set'
+
+  - task: powershell@2
+    inputs:
+      targetType: filePath
+      filePath: build\Install-WindowsSdkISO.ps1
+      arguments: 17763
+    displayName: 'Install RS5 SDK (17763)'
+
+  - task: NuGetToolInstaller@0
+    displayName: 'Use NuGet 4.9.1'
```

Comment:
>4.9.1 [](start = 28, length = 5)

I think 4.9.1 and 17763 above should be variables. and it's better to keep all variables to one file

Reply:
This yaml syntax is not well documented and I can't find an easy way to re-use things. Let me know if you find otherwise in the azure pipeline docs.

Reply:
Maybe it's over design, another way is put it in scripts like Install-WindowsSdkISO.ps1 and NugetToolInstall.ps1.
Then the scripts to parse and get the correct parameter from one file.

---
In reply to: [241884066](https://github.com/Microsoft/microsoft-ui-xaml/pull/87#discussion_r241884066) [](ancestors = 241884066)